routers/router.php

```diff
@@ -188,6 +189,10 @@ public function send_command($command, $parameter) {
       case 'zebra':
         return new Quagga($config, $router_config, $id, $requester);
 
+      case 'frr':
+      case 'zebra':
```

Comment:
This will conflict with the first `zebra` case, which use the `Quagga` class.

Reply:
Must be resolve now